pkg/ccl/backupccl/backupdest/incrementals.go

```diff
@@ -32,21 +32,41 @@ const (
 // backupSubdirRE identifies the portion of a larger path that refers to the full backup subdirectory.
 var backupSubdirRE = regexp.MustCompile(`(.*)/([0-9]{4}/[0-9]{2}/[0-9]{2}-[0-9]{6}.[0-9]{2}/?)$`)
 
-// CollectionAndSubdir breaks up a path into those components, if applicable.
-// "Specific" commands, like BACKUP INTO and RESTORE FROM, don't need this.
-// "Vague" commands, like SHOW BACKUP and debug backup, sometimes do.
-func CollectionAndSubdir(path string, subdir string) (string, string) {
```

Comment:
nit: I think it's worth mentioning in the comment above or inline that a subdir will always get passed for new style SHOW BACKUP.